### PR TITLE
Mod 4 Links and Calendar

### PR DESCRIPTION
**Pull request by memcmahon**
This **will** cause merge conflicts with open PRs, but I think let's push this up and the conflicts will hopefully help us add new links as they are available.

### CODE REVIEW

File: module4/index.md [hunk 1]
```diff
@@ -16,15 +16,26 @@ In Module 4, students will begin to dive into the skills and mindsets necessary
```
Comment (zoefarrell) [inline module4/index.md:16]: I think the subheading and Overview here are leftover from mod 1?

File: module4/index.md [hunk 2]
```diff
@@ -16,15 +16,26 @@ In Module 4, students will begin to dive into the skills and mindsets necessary
 
 ## Curriculum
 
-< Under Construction >
-<!-- 
-
 ### Week 1
```
Comment (zoefarrell) [inline module4/index.md:21]: Can you also add the intermission work in this Pr?

**Review by zoefarrell — APPROVED** [review]

**Comment by zoefarrell** [comment]
> This **will** cause merge conflicts with open PRs, but I think let's push this up and the conflicts will hopefully help us add new links as they are available.

I agree, let's rip the merge conflict bandaid off. They should be pretty clear how to resolve.

File: module4/index.md (hunk 2 above)
Comment (memcmahon) [reply module4/index.md:21]: Oh! Yes, definitely

**Pull request merged**